flixel/addons/display/FlxStarField.hx

```diff
@@ -106,8 +105,17 @@ class FlxStarField3D extends FlxStarField
 private class FlxStarField extends FlxSprite
 {
 	public var bgColor:Int = FlxColor.BLACK;
-	
-	private var _stars:Array<FlxStar>;
+
+	// Why five Vectors instead of one that has an FlxStar class?  This is better
+	// for the cache, as all the data is being accessed sequentially so it'll be a
+	// little faster.
+	private var _starsX:Vector<Float>;
```

Comment:
Nothing formal, I just ran Flixius twice (before and after the changes) and looked at the update times.  What would you suggest I do for a more formal benchmark?